### PR TITLE
N8N-4255: Remove request libraries from cli package

### PR DESCRIPTION
**Pull request by brianinoa**
https://linear.app/n8n/issue/N8N-4255/remove-from-cli-package

### CODE REVIEW

**Review by krynble — CHANGES_REQUESTED** [review]

File: packages/cli/src/Server.ts [hunk 1]
```diff
@@ -793,11 +785,11 @@ class App {
 							400,
 						);
 					}
-					const data = await requestPromise.get(req.query.url as string);
+					const { data } = await axios.get<IWorkflowResponse>(req.query.url as string);
 
 					let workflowData: IWorkflowResponse | undefined;
 					try {
-						workflowData = JSON.parse(data);
+						workflowData = data;
```
[the remark this answers is not in the document]
Comment (brianinoa) [reply packages/cli/src/Server.ts:792]: @krynble 👍 Will do that immediately

**Review by krynble — APPROVED** [review]
Nice thanks!

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.189.0`

**Pull request merged**